package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bellawatt/electric-rate-engine",
-  "version": "2.0.1",
+  "version": "3.0.0",
```

Comment:
one more thing... can you remove this change? the NPM CLI has a command that will update this and create a tag, which is nice to have